test/integration/mjolnirSetupUtils.ts

```diff
@@ -82,6 +82,7 @@ export async function makeMjolnir(): Promise<Mjolnir> {
     LogService.info("test/mjolnirSetupUtils", "Starting bot...");
     const pantalaimon = new PantalaimonClient(config.homeserverUrl, new MemoryStorageProvider());
     const client = await pantalaimon.createClientWithCredentials(config.pantalaimon.username, config.pantalaimon.password);
+    await overrideRatelimitForUser(await client.getUserId());
```

Comment:
I thought we were *not* overriding the rate limit for Mjölnir itself?

Reply:
We are and it's problematic not to because all of the other tests use it

Reply:
If we override the rate limit for the Mjölnir user, we're not testing the backoff-and-retry, right?

Reply:
we are, there is a dedicated test https://github.com/matrix-org/mjolnir/blob/0c68d89b5e788ba8a6bf2f78483c5499ab9cc671/test/integration/throttleTest.ts

Reply:
Sorry, I meant we're not testing *Mjölnir* with the backoff-and-retry. We're testing it as unit test, not as integrated in Mjölnir. Right?

Reply:
Yes, but there should not be any difference since all requests share this same patch and using Mjolnir will put the limiter under less stress than this test does. It might be a good idea for "if things change" in case there will be a difference, but I'm not sure. I think it's a really bad idea to rate limit all the tests because it complicates what each test is supposed to test and it makes things slow and will cause us to avoid running them. If we want to test Mjolnir itself with rate limits we could start syncing a new instance in that test with its own management room and perform a redaction large or something, but I don't see a lot of value in it.

Reply:
How bad is the effect on runtime if we're testing Mjölnir with the backoff-and-retry (and everything else unthrottled)?

Reply:
Enough to make some tests need a longer timeout, but seems to be a few (2-5)s per test

Reply:
I have added a dedicated test for Mjolnir syncing and responding to commands because then any issue with rate limiting will be isolated

Reply:
Ok, let's try that.